lib/pages/people/view/person_view.dart

```diff
@@ -120,3 +35,97 @@ class PersonView extends StatelessWidget {
     );
   }
 }
+
+class _PersonViewState extends State<PersonView> {
+  final ScrollController _controller = ScrollController();
+  
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Container(
+        color: Theme.of(context).scaffoldBackgroundColor,
+        // TODO(RavanRotaru): extract scrollbar in another widget
+        child: Scrollbar(
+          thickness: 10,
+          isAlwaysShown: true,
+          controller: _controller,
+          child: SingleChildScrollView(
+            controller: _controller,
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(30, 30, 30, 5),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.start,
+                    children: [
+                      const Expanded(
```

Comment:
Those Expanded SizedBoxes are ok for the mock.
In the final version we will have everything in the web constrained to be centered and fit a maximum width, therefore, those extra pseudo alignments won't be needed.

Reply:
Now all widgets are centered without Expanded SizedBox [here](https://github.com/student-hub/acs-upb-mobile/blob/396dff90559907416f9ea31238fb87d37c747e59/lib/pages/people/view/person_view.dart#L49-L59). But there is a problem when the web page is resized.


![Untitled](https://user-images.githubusercontent.com/68648367/132522962-7e693654-ba38-4f8d-9c4e-dde790037642.png)

Reply:
maybe find a solution for the overflowing text

Reply:
fixed here https://github.com/student-hub/acs-upb-mobile/pull/252/commits/e6472af39ea8bf49f03a55570c4b7cdf661d2833